### PR TITLE
Code style change

### PR DESCRIPTION
**Pull request by Tristan22400**
fix: https://github.com/morpho-dao/morpho-data-structures/issues/125

### CODE REVIEW

**Comment by MathisGD** [comment]
This PR needs a rebase on #44

**Review by pakim249CAL — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]
I wouldn't have done the changes on the test but LGTM

File: test/mocks/HeapOrderingMock.sol [hunk 1]
```diff
@@ -1,55 +1,59 @@
...
-import "src/HeapOrdering.sol";
-import "./IHeapOrderingMock.sol";
+import {HeapOrdering} from "src/HeapOrdering.sol";
+import {IHeapOrderingMock} from "./interfaces/IHeapOrderingMock.sol";
 
 contract HeapOrderingMock is IHeapOrderingMock {
     using HeapOrdering for HeapOrdering.HeapArray;
 
-    HeapOrdering.HeapArray internal heap;
+    /* STORAGE */
```
[the remark this answers is not in the document]
Comment (Tristan22400) [reply test/mocks/HeapOrderingMock.sol:10]: do you want to revert the changes?

[the remark this answers is not in the document]
Comment (MerlinEgalite) [reply test/mocks/HeapOrderingMock.sol:10]: If you have time yes please

**Review by MerlinEgalite — APPROVED** [review]

**Review by pakim249CAL — DISMISSED** [review]

**Review by makcandrov — APPROVED** [review]

File: test/TestCommonHeapOrdering.t.sol [hunk 2]
```diff
@@ -1,8 +1,9 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.0;
 
-import "src/HeapOrdering.sol";
-import "./helpers/RandomHeap.sol";
+import {RandomHeap} from "./helpers/RandomHeap.sol";
+import {HeapOrdering} from "src/HeapOrdering.sol";
+import {HeapOrderingMock} from "./mocks/HeapOrderingMock.sol";
```
Comment (mlgarchery) [inline test/TestCommonHeapOrdering.t.sol:6]: unused

File: test/TestThreeHeapOrderingInvariant.t.sol [hunk 3]
```diff
@@ -49,8 +49,8 @@ contract TestThreeHeapOrderingInvariant is Test {
 
         for (uint256 i; i < length; ++i) {
             assertTrue((i * 3 + 1 >= length || i * 3 + 1 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 3 + 1)));// forgefmt: disable-line
-            assertTrue((i * 3 + 2 >= length || i * 3 + 2 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 3 + 2)));// forgefmt: disable-line
-            assertTrue((i * 3 + 3 >= length || i * 3 + 3 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 3 + 3)));// forgefmt: disable-line
+            assertTrue((i * 3 + 2 >= length || i * 3 + 2 >= heap.size()|| heap.accountsValue(i) >= heap.accountsValue(i * 3 + 2)));// forgefmt: disable-line
+            assertTrue((i * 3 + 3 >= length || i * 3 + 3 >= heap.size()|| heap.accountsValue(i) >= heap.accountsValue(i * 3 + 3)));// forgefmt: disable-line
         }
```
Comment (mlgarchery) [inline test/TestThreeHeapOrderingInvariant.t.sol:54]: is the spaces removal wanted ?

**Review by MerlinEgalite — APPROVED** [review]

**Review by pakim249CAL — APPROVED** [review]

**Pull request merged**